docker-php-ext-install

```diff
@@ -126,7 +126,7 @@ for ext in $exts; do
 	find modules \
 		-maxdepth 1 \
 		-name '*.so' \
-		-exec basename '{}' ';' \
+		-exec basename '{}' '.so' ';' \
```

Comment:
Can you elaborate on this (seemingly unrelated) change?  The `.so` vs not-`.so` is already handled in `docker-php-ext-enable` unless I've missed something.